tests/foreman/api/test_activationkey.py

```diff
@@ -5,7 +5,11 @@
 from nailgun import client, entities
 from requests.exceptions import HTTPError
 from robottelo.config import settings
-from robottelo.datafactory import valid_data_list
+from robottelo.datafactory import (
+    invalid_names_list,
+    valid_data_list,
+    valid_names_list,
```

Comment:
Please use valid_data_list instead of valid_names_list in activation key tests.  No other tests are using valid_names_list for now.  I may remove or modify this function soon.